go.mod

```diff
@@ -89,3 +89,10 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.2 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace (
+	github.com/NVIDIA/nvidia-container-toolkit => ../container-toolkit
```

Comment:
...?

Reply:
This was to "preview" the required changes since I was not able to pull in the SHA from upstream. I have since created a tag and referenced that.